### DISCUSSION TITLE
Proxmox Kernel Selector

### DISCUSSION THREAD

**Discussion by tworth04** [tteck/Proxmox#3084] [Ideas]
I know we have kernel pinning, but could we have an option for selecting a specific kernel?  I just upgraded my server and started fresh and I don't have the option to pin the current 6.5 kernel.  I only have 6.8 available.

**Comment by tteck** [comment]
That's because Proxmox VE 8.2 ships with only the 6.8 kernel. You will have to manually install the 6.5 kernel if needed.

**Reply by tworth04** [reply]
understood that...Was thinking something along the lines of a helper for the kernel pin script.  Sorry if i wasn't clear on that.  Appreciate you work

**Comment by bobloadmire** [comment]
So i'm also trying to pin kernel 6.5, however I don't have it installed for whatever reason. So running apt search pve-kernel lists all available, but i'm only seeing 6.2 as an option. How do I install 6.5?

**Reply by thijspomper** [reply]
try to apt search "proxmox-kernel"
